### PR TITLE
Add support for serviceAccount, workingDir, command and args.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

**Comment by iocanel** [comment]
This is broken, I will close and reopen when issues are addressed.

**Comment by geoand** [comment]
So should I ignore the review request for the time being?

File: core/src/main/java/io/ap4k/kubernetes/decorator/AbstractAddProbeDecorator.java [hunk 1]
```diff
@@ -30,15 +30,16 @@
 import java.util.Arrays;
 import java.util.Collections;
 
-public abstract class AbstractAddProbeDecorator extends Decorator<ContainerBuilder> {
+public abstract class AbstractAddProbeDecorator extends ApplicationContainerDecorator {
```
Comment (geoand) [inline core/src/main/java/io/ap4k/kubernetes/decorator/AbstractAddProbeDecorator.java]: Could we perhaps have a small comment on what the point of the class is?

**Comment by geoand** [comment]
@iocanel Maybe also update some of the examples to showcase the new features?

File: core/src/main/java/io/ap4k/kubernetes/decorator/AbstractAddProbeDecorator.java (hunk 1 above)
Comment (iocanel) [reply core/src/main/java/io/ap4k/kubernetes/decorator/AbstractAddProbeDecorator.java]: ApplicationContainerDecorator does have a comment.
Added a comment on AbstractAddProbeDecorator.

**Comment by iocanel** [comment]
@geoand: For the time being I don't have a use case common enough that would justify an example.

**Pull request merged**